### PR TITLE
Subject specific maths page

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/hC7mG9vN/2694-create-new-design-for-subject-specific-pages

### Context

This PR is intended to implement some reusable components that will initially be used for subject-specific content.

The longer-term intention is to use them to replace some of the existing styles on the site so the design language is more consistent throughout.

#### Header

The header will fit where the current hero component does, it will contain some header text and an image. A `colour` argument allows it to be styled per-page; here's some examples:

| Blue/Pink | Yellow/Purple | Pink/Yellow |
| ----- | ------ | ------|
| Screenshot from 2021-12-13 13-48-22 | Screenshot from 2021-12-13 13-48-28 | Screenshot from 2021-12-13 13-48-25 |
| Screenshot from 2021-12-14 16-49-22 |Screenshot from 2021-12-14 16-49-54 | Screenshot from 2021-12-14 16-49-42 |
| Screenshot from 2021-12-14 16-50-48 | Screenshot from 2021-12-14 16-50-17 | Screenshot from 2021-12-14 16-50-32 |

#### Previews

* ~~[English](https://review-get-into-teaching-app-2099.london.cloudapps.digital/subjects/english)~~
* [Maths](https://review-get-into-teaching-app-2099.london.cloudapps.digital/subjects/maths)
* ~~[Science](https://review-get-into-teaching-app-2099.london.cloudapps.digital/subjects/science)~~

### CODE REVIEW

**Comment by peteryates** [comment]
Added the maths content, page is looking good!

File: app/views/content/subjects/maths/_article.html.erb [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+          tempor incididunt ut labore et dolore disputandum putant. Sed ut omittam
+          pericula, labores, dolorem etiam, quem optimus quisque pro patria et pro
+          suis.
+        </p>
+      </blockquote>
+
+      <figcaption>Quoniam Nacti</figcaption>
+    </figure>
+  </section>
```
Comment (peteryates) [inline app/views/content/subjects/maths/_article.html.erb:73]: @EllieNodder - I added a placeholder for a quote, please feel free to make an edit if you find one :)

**Comment by SaimaBodiat** [comment]
Hi, 

- just wondered if it was the result of font style which means the letters S and M in 'Science' and 'Maths' don't look obviously capitalised in the same way 'English' does?

- The internship paragraph states 'find out more'  which isn't accessible, would be able to change that?

Thanks

**Comment by peteryates** [comment]
>  * just wondered if it was the result of font style which means the letters S and M in 'Science'
>  and 'Maths' don't look obviously capitalised in the same way 'English' does?

That's just because English is a proper noun - all the language subjects will be capitalised like this.

>  * The internship paragraph states 'find out more'  which isn't accessible, would be able to change that?

Ah well spotted, fixed this in the latest push

**Comment by Tomcowan1** [comment]
Hi @peteryates - potentially dumb Q, but where did we put this in the nav - struggling to find on the review app.

**Comment by peteryates** [comment]
@Tomcowan1 - it's not in the nav. Realistically I think we need to drop something else (stories) before we add it.

**Comment by peteryates** [comment]
Ah sorry! If you look at the bottom of the PR there's page-specific links. We dropped science and English for the time being (used for testing out colour schemes) but the maths one should work

**Comment by Tomcowan1** [comment]
Ok thanks. I can't see the changes I've made this morn, but think that's because merging is blocked for "Error: Process completed with exit code 1." Is that an easy fix?

It might be the maths internship section - it can't be in there for live because the programme hasn't been set up yet.

**Comment by peteryates** [comment]
Yeah, just a linting error (whitespace at end of line). Will fix now

**Review by gemmadallmandfe — APPROVED** [review]
Looks great - happy for this to be merged first thing tomorrow (won't merge tonight given issues with PRs getting stuck)

**Comment by gemmadallmandfe** [comment]
@Tomcowan1 Could you just add the policy tag and a note to say this has been through policy/curriculum (as appropriate)? Thanks!

**Comment by Tomcowan1** [comment]
Policy sign off, quoted maths teacher, and information about importance of maths to the economy, come from the Maths Curriculum Policy Lead.

**Pull request merged**